src/NzbDrone.Core/Notifications/Discord/Discord.cs

```diff
@@ -317,6 +317,16 @@ public override void OnApplicationUpdate(ApplicationUpdateMessage updateMessage)
                                           {
                                               Name = "New Version",
                                               Value = updateMessage.NewVersion.ToString()
+                                          },
+                                          new DiscordField()
+                                          {
+                                              Name = "New",
```

Comment:
These could get to long for a discord message couldn't they? I cannot remember the character limit but a master release as an example could be a big release with lots of new and fixed

Reply:
Yea, Probably should do a take 5 or something

Reply:
> Yea, Probably should do a take 5 or something

take 5 and a link to github for further info?

Reply:
Personally i build the entire thing, truncate > 2000 (1000 for new & 1000 for fixed) so as much as possible will fit incase there are a lot of short messages so more fit. Just as a thought.